src/App.js

```diff
@@ -17,10 +17,9 @@ const Body = styled.div`
   align-items: center;
 `;
 
-const siteUrl = 'https://icecast.softboys.club:18000';
-const ssl = (listenurl) => {
-  let trackpath = !listenurl ? null : listenurl.slice(listenurl.lastIndexOf('/'));
-  return siteUrl + trackpath;
+const sslUrl = (trackUrl) => {
```

Comment:
- blocker
need to update the function calls after changing the function name

on line  36/37 ssl is undefined.